### ISSUE TITLE
some training questions about simultaneous segmentation of multiple lesions

### ISSUE THREAD

**Issue by cloudyliy** [csyizhou/FGADR#3]
I recently read your article "A Benchmark for Studying Diabetic Retinopathy: Segmentation, Grading, and Transferability" published in TMI. Regarding task 1, segmentation of multiple fundus lesions, I used unet to segment the lesions on the FGADR dataset, and the average AP on EX, HE, MA, and SE can only reach 0.4. But I see the effect in your article is very good, only using unet for segmentation, the average AP can reach 0.6. Is there a way to improve the effect in the segmentation task? Can you tell us about the image preprocessing method and the specific loss function, the number and type of GPU used in order to reproduce the result?

**Comment by qingliucs** [comment]
Hi, have you reproduced the result?

**Comment by osivaz61** [comment]
I am getting the following dice score results:

MA : 23.81
HE  : 60.37
EX  : 63.19
SE  : 51.26

There is a huge gap for MA and SE lesions. Is there anyone who gets better or the same results in the paper?
Thanks

**Comment by ParadiseforAndaChen** [comment]
Hi I have send the author the request for the data but get no answer. How could I get the access of the data, which is of great importance for our study. Thanks a lot.

**Comment by moha23** [comment]
> Hi I have send the author the request for the data but get no answer. How could I get the access of the data, which is of great importance for our study. Thanks a lot.

Same here, no response yet, were you able to get the data? @ParadiseforAndaChen

**Comment by Livingyang002** [comment]
> > Hi I have send the author the request for the data but get no answer. How could I get the access of the data, which is of great importance for our study. Thanks a lot.
> 
> Same here, no response yet, were you able to get the data? @ParadiseforAndaChen
I haven't received a reply yet. Have you received the data? How long will it take to get a reply? @ParadiseforAndaChen @moha23

**Comment by esterlee27** [comment]
> > > Hi I have send the author the request for the data but get no answer. How could I get the access of the data, which is of great importance for our study. Thanks a lot.
> > 
> > 
> > Same here, no response yet, were you able to get the data? @ParadiseforAndaChen
> > I haven't received a reply yet. Have you received the data? How long will it take to get a reply? @ParadiseforAndaChen @moha23

Hi everyone, I’ve also applied for access to the dataset but haven’t received a reply yet. Have any of you received a response? If so, how long did it take? Thanks for any updates!

**Comment by moha23** [comment]
> > > > Hi I have send the author the request for the data but get no answer. How could I get the access of the data, which is of great importance for our study. Thanks a lot.
> > > 
> > > 
> > > Same here, no response yet, were you able to get the data? @ParadiseforAndaChen
> > > I haven't received a reply yet. Have you received the data? How long will it take to get a reply? @ParadiseforAndaChen @moha23
> 
> Hi everyone, I’ve also applied for access to the dataset but haven’t received a reply yet. Have any of you received a response? If so, how long did it take? Thanks for any updates!

Hey, yes, the authors did share part of the data, took about 2 weeks for me.

**Comment by tonyaraque** [comment]
> I am getting the following dice score results:
> 
> MA : 23.81 HE : 60.37 EX : 63.19 SE : 51.26
> 
> There is a huge gap for MA and SE lesions. Is there anyone who gets better or the same results in the paper? Thanks
@osivaz61 Could you explain how you achieved EX=63.19? I haven't been able to get past 0.56 (Macro Dice) or 0.52 (mDice).
